tests/Object/RouteTest.php

```diff
@@ -31,6 +31,7 @@ public function testSuccessfulRouteNotDirect()
 		$response = $this->router->route('app/TestController/test', [], false);
 
 		$this->assertNotError();
```

Comment:
could be better to not have a default object in `assertNotError`. It would make reading the test a little more straightforward. I had to dig to see that if an object is not given it checks the router. :shrug:

Reply:
Fair point. it's beyond this scope, but would make maintaining the framework easier.